### DISCUSSION TITLE
A case for FScope/DSO-500 boards

### DISCUSSION THREAD

**Discussion by gilesp1729** [fhdm-dev/scoppy#83] [Show and tell]
[Here's a 3D printed case](https://www.myminifactory.com/object/3d-print-251021) for these boards that I made. It has enough height to accommodate a Pico up on headers or soldered to the board, and also access to the pins and jumpers.

(For some reason the website won't allow 3D visualisations of the STL files, although they download OK.)
![2022-10-18 20 49 11](https://user-images.githubusercontent.com/21351326/197375683-55356a7c-6056-4c3c-9353-0e2e729266d3.jpg)
![2022-10-18 20 49 58](https://user-images.githubusercontent.com/21351326/197375685-6b0fe34d-0122-4ef9-8198-c05fea77414e.jpg)

**Comment by odysseus55** [comment]
Very nice job.
Now I have to buy a 3D printer X(
